_data/sites.json

```diff
@@ -5223,9 +5217,9 @@
 
     {
         "name": "RunKeeper",
-        "url": "http://runkeeper.com/I-Love-RunKeeper",
-        "difficulty": "medium",
-        "notes": "After clicking the button, you will have to enter your password, complete a capcha, click another button, AND a confirmation modal dialog.",
+        "url": "https://runkeeper.com/delete-account?confirm",
+        "difficulty": "easy",
+        "notes": "Enter your password, select a reason for deletion, complete the captcha, press the delete button and then confirm it in the dialog.",
         "notes_it": "Dopo aver cliccato il bottone dovrai: inserire la tua password, completare un capcha, clicare un altro bottone E un dialogo di conferma.",
```

Comment:
Since the English notes have changed, the translations should be updated or removed.